private/buf/bufwork/workspace_builder.go

```diff
@@ -56,6 +56,9 @@ func (w *workspaceBuilder) BuildWorkspace(
 	if workspaceConfig == nil {
 		return nil, errors.New("received a nil workspace config")
 	}
+	if configOverride != "" {
+		return nil, errors.New("the --config flag has been disabled for workspaces")
```

Comment:
I'd update the message a bit here - we shouldn't refer to it as something that was previously enabled.
```suggestion
		return nil, errors.New("the --config flag has is not compatible with workspaces")
```

Reply:
Ah yeah, that is a lot cleaner, I am a fan of this :) (Committing this suggestion, I know it'll break tests, lol, I'll fix that, haha)